### PR TITLE
log epoch number for tensorboard

### PR DESCRIPTION
**Pull request by charlieyou**

### CODE REVIEW

File: python/mxnet/contrib/tensorboard.py [hunk 1]
```diff
@@ -70,4 +70,4 @@ def __call__(self, param):
         for name, value in name_value:
             if self.prefix is not None:
                 name = '%s-%s' % (self.prefix, name)
-            self.summary_writer.add_scalar(name, value)
+            self.summary_writer.add_scalar(name, value, global_step=param.epoch)
```
Comment (zihaolucky) [inline python/mxnet/contrib/tensorboard.py]: What would happened if we use this in batch_end_callback?

**Comment by zihaolucky** [comment]
I've checked this change. LGTM.

Notice that, after we add `global_step`:

 * In `RELATIVE` mode, the training curve still logs the metric every batch, the curve looks more smooth. Same with the older version, great.
 * In `STEP` mode, it shows the metric in that epoch.

**Comment by szha** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to ping me to reopen if this is still an active issue. Thanks!
Also, do please check out our [forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) for general "how-to" questions.

**Comment by charlieyou** [comment]
@szha Completely forgot about this, sorry! Would like to re-open. Very small change, anything else needed from me to merge this?

**Pull request closed without merging**